doc/source/CHANGELOG.rst

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented here.
 
 The format is largely inspired by keepachangelog_.
 
+v0.8.2 - Unreleased
```

Comment:
Why patch the patch version if this isn't a bugfix. Shouldn't we patch de minor?